otp_build

```diff
@@ -1017,7 +1041,7 @@ do_debuginfo_win32 ()
     fi
     BINDIR="$ERL_TOP/bin/$TARGET"
     EVSN=`grep '^VSN' erts/vsn.mk | sed 's,^VSN.*=[^0-9]*\([0-9].*\)$,@\1,g;s,^[^@].*,,g;s,^@,,g'`
-    for f in beam.debug.smp.dll beam.smp.pdb beam.debug.smp.dll.pdb erl.pdb erlexec.pdb; do
+    for f in beam.debug.smp.dll beam.smp.pdb beam.debug.smp.dll.pdb werl.pdb erl.pdb erlexec.pdb; do
```

Comment:
I guess werl.pdb is not needed anymore since werl is just a copy of erl?

Reply:
Okay, I'll remove it!